### DISCUSSION TITLE
Device Profiles?

### DISCUSSION THREAD

**Discussion by huntingtontown** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#89] [Q&A] [answered]
New to the Github community here, howdy y'all. Read through the manual tonight again... didn't see anything about Device Profiles. Searched but didn't find anything in the Facebook group about it recently, and see nothing about it on here. What am I missing? Was there a decision made to go in a different direction?

**Comment by simonaglover95** [comment, marked as answer]
Hey there :) Not at all! If (as I think you are) you're referring to the Device Library, it's part of the web editor which is only a few short days from being released for public use! 

The Device Library is a database of the MIDI Commands that manufacturers have given their devices as factory-standard. Some devices which ARE MIDI capable, may not have factory defaults and require the user to configure which CC is assigned to which parameter and so on. This is usually on the case for more complex digital systems like AXE-FX III. 

However, in terms of a whole bank or multi-bank preset for all sorts of pedal controls, that will be possible with the"export to file" feature in the web editor and the "create from template" feature. At launch of the web editor, this will still be in its infancy, but there will be more features added to it along the way to make it easier and easier to get started really quickly. 

For now, it is possible to import the settings from your device, or start from a blank template. You can make any config you like, including using the Device Library to your heart's content, and then either send it all to the BRIDGE controller, or save it to your PC as a template file (.json). 
These files can easily be shared online, and we encourage it! 
In the future, we plan to add some of these config files to our list of templates which can help people get started with a particular goal in mind, and skip a lot of the setup work. 

I hope this answers you question! and as always, we're happy to hear your feedback 👍 

Simon
